tests/unit-poet.yaml

```diff
@@ -30,6 +30,6 @@ services:
     working_dir: '/project/sawtooth-poet/src/core'
     entrypoint: "bash -c \"\
         source /project/sawtooth-poet/build_sgx_binary.sh \
-        && cargo test \""
+        && cargo test -- --test-threads=1\""
```

Comment:
can we execute test cases concurrently. If enclave termination is causing issue we could probably avoid running that specific test case